pkg/util/k8sutil/k8sutil.go

```diff
@@ -0,0 +1,127 @@
+package k8sutil
```

Comment:
I suggest we also named it to `kube` like Helm. https://github.com/helm/helm/tree/master/pkg/kube